packages/main/src/Token.ts

```diff
@@ -0,0 +1,249 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+import slot from "@ui5/webcomponents-base/dist/decorators/slot.js";
+import event from "@ui5/webcomponents-base/dist/decorators/event.js";
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+import languageAware from "@ui5/webcomponents-base/dist/decorators/languageAware.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import { getTheme } from "@ui5/webcomponents-base/dist/config/Theme.js";
+import {
+	isBackSpace,
+	isSpace,
+	isDelete,
+	isSpaceCtrl,
+} from "@ui5/webcomponents-base/dist/Keys.js";
+import "@ui5/webcomponents-icons/dist/decline.js";
+import "@ui5/webcomponents-icons/dist/sys-cancel.js";
+import type { ITabbable } from "@ui5/webcomponents-base/dist/delegate/ItemNavigation.js";
+import type I18nBundle from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import { getI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import { TOKEN_ARIA_DELETABLE } from "./generated/i18n/i18n-defaults.js";
+
+import Icon from "./Icon.js";
+import TokenTemplate from "./generated/templates/TokenTemplate.lit.js";
+
+// Styles
+import styles from "./generated/themes/Token.css.js";
+
+type TokenDeleteEventDetails = {
+	"backSpace": boolean;
```

Comment:
you can declare `delete` and `backSpace` without double quotes

Reply:
Done